src/SDK/Trace/ExporterFactory.php

```diff
@@ -83,24 +84,24 @@ public function fromEnvironment(): ?SpanExporterInterface
         }
         $exporter = $exporters[0];
         switch ($exporter) {
-            case 'none':
+            case Values::VALUE_NONE:
                 return null;
-            case 'jaeger':
-            case 'zipkin':
-            case 'newrelic':
+            case Values::VALUE_JAEGER:
+            case Values::VALUE_ZIPKIN:
+            case Values::VALUE_NEWRELIC:
             case 'zipkintonewrelic':
```

Comment:
is there a reason this one was left untouched?

Reply:
The refactoring has been done for those variable which are already defined in the knownvalues inteface. 'zipkintonewrelic' is not defined there. The knownvalues interface only has variables which are defined in the OTEL spec -  https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#exporter-selection. That's why it has been ommitted.

Reply:
@kpratyus 
Ok, got you.
I did not remember, that I didn't add those values.
In general our "known values" could have more values than the ones stated in the spec.
However the exporter factory is subject to change in this case anyway as we may have to deal
with more vendor specific exporters in general, so we must have a more dynamic way than hard coding
all the options. Also actually users should be able to use thier custom exporters.

@brettmc 
So, I think we should just merge this as is, and take care of tis later (I ill keep in mind)
Do you agree?

Reply:
Yes, I'm happy to merge as is